web/package.json

```diff
@@ -12,6 +13,7 @@
     "postcss-import": "^14.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-preloaders": "^3.0.3",
```

Comment:
Nope, we would rather user a customer preload

Reply:
ok